mysite/polls/models.py

```diff
@@ -69,3 +44,49 @@ def create_superuser(
         user.is_staff = True
         user.save()
         return user
+
+
+class Question(models.Model):
+    question_text = models.CharField(max_length=200)
+    pub_date = models.DateTimeField("date published")
+    created_by = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.question_text
+
+    def was_published_recently(self):
+        now = timezone.now()
+        return now - datetime.timedelta(days=1) <= self.pub_date <= now
+
+    was_published_recently.admin_order_field = "pub_date"
+    was_published_recently.boolean = True
+    was_published_recently.short_description = "Published recently?"
```

Comment:
That is true, better make a better use of the Lookup API. Will propose changes later today.

Reply:
You know what? I noticed that the **was_published_recently** method is only used in the Django admin site as an example of boolean fields. But in the project as a whole, it is not used at all, no <Model>ViewSet nor function ApiView depends on it, much less sorting or filtering with it. I propose a deletion of the usage of the method in the admin site. Maybe we shoukd keep the method around in case it can be useful at a later occasion.

Reply:
Yes that should work